src/arch/traits.rs

```diff
@@ -13,6 +29,12 @@ use crate::internal::BeBytes;
 // TODO: add (optional?) trait methods for reading/writing specific register
 // (via it's GDB index)
 pub trait Registers: Default {
+    /// Register identifier for addressing single registers.
+    ///
+    /// If your target does not implement that feature, you can use `RegId = ()`
```

Comment:
This doc comment needs to be reworded.

- Avoid using "you"/"your" when possible.
- The end-user target has nothing to do with `RegId`, it's up to the `Arch` author to include a valid RegID.
- Add a note mentioning that `()` is a _temporary_ measure, and that all newly added architectures are required to provide a register identifier
    - Additionally, it would be nice to add a brief note to nudge people towards opening a PR if they see `RegID = ()` :smile: